babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ["@babel/preset-env"],
+};
```

Comment:
This was needed for running the jest tests otherwise it complains about using imports in a non-module or something like that